libs/features/placeholders.js

```diff
@@ -69,7 +69,7 @@ async function getPlaceholder(key, config, sheet) {
     return defaultPlaceholders;
   });
 
-  if (placeholders?.[key]) return placeholders[key];
+  if (typeof placeholders?.[key] !== 'undefined') return placeholders[key];
```

Comment:
Placeholder values should be strings, regardless of their length, so would it make sense to instead check for `typeof placeholders?.[key] === 'string'`?

Reply:
I was wondering the same, I'm not 100% certain on how EDS handles this and I _think_ values might be `null` which would still mean the key exists but the value is omitted